### ISSUE TITLE
Request: Mutator for random powerups

### ISSUE THREAD

**Issue by HellDragon-HK** [GreatEmerald/UT3Weapons#2]
- [ ] UT3 has several powerups that UT2004 doesn't or is found in adrenaline form, I was wondering if it would be possible to create a mutator to utilize these powerups by randomly cycling the UDamage pickup
 - [ ] Could the same thing be done with special item pickups like spider traps, emp mines, and slow fields?

**Comment by GreatEmerald** [comment]
Yes. Effectively it would be a matter of replacing UDamage bases with Wildcard bases, it's not very difficult.
